doc/sphinx/writing-tactics.rst

```diff
@@ -0,0 +1,35 @@
+.. _writing-tactics:
+
+===============
+Writing tactics
+===============
+
+The languages presented in this chapter allow one to build complex
+tactics by reusing and combining existing ones.  While :ref:`Ltac
+<ltac>` was initially thought of as a language for doing some basic
+combinations, it has been used successfully to build highly complex
+tactics as well, but this has also highlighted its limits and
+fragility.  The experimental language :ref:`Ltac2 <ltac2>` is a typed
+and more principled variant which is more adapted to building complex
+tactics.
+
+.. toctree::
+   :maxdepth: 1
+
+   proof-engine/ltac
+   proof-engine/ltac2
+
+
+Other solutions exist beyond these two tactic languages to write new
```

Comment:
```suggestion
Another alternative tactic languages is Mtac2:
```

Reply:
Alone, this suggestion kind of kills the idea of this sentence introducing a bullet point list of several alternative solutions.

Reply:
OK, but I suggest this wording: "There are other solutions beyond ..."